code/modules/projectiles/updated_projectiles/guns/rifles.dm

```diff
@@ -137,71 +137,63 @@
 
 
 //-------------------------------------------------------
-//MAR-40 AK/FN FAL clone
-
-/obj/item/weapon/gun/rifle/mar40
-	name = "\improper MAR-40 battle rifle"
-	desc = "A cheap, reliable assault rifle chambered in 7.62x39mm. Commonly found in the hands of criminals or mercenaries."
-	icon_state = "mar40"
-	item_state = "mar40"
-	caliber = "7.62x93mm" //codex
+//Literally just the AK-47
```

Comment:
Just remove it, honestly. The purpose of comments is to explain why you are doing something code-wise, and I think just about everyone can make out that  the gun named "AK-47" is the AK-47

Reply:
```suggestion
```